airbyte-cdk/python/airbyte_cdk/sources/file_based/config/abstract_file_based_spec.py

```diff
@@ -51,40 +50,21 @@ def replace_enum_allOf_and_anyOf(schema: Dict[str, Any]) -> Dict[str, Any]:
         anyOfs are also not supported by the UI, so we replace them with the similar oneOf, with the
         additional validation that an incoming config only matches exactly one of a field's types.
         """
-        # this will need to add ["anyOf"] once we have more than one format type and loop over the list of elements
         objects_to_check = schema["properties"]["streams"]["items"]["properties"]["format"]
-        if "additionalProperties" in objects_to_check:
-            objects_to_check["additionalProperties"]["oneOf"] = objects_to_check["additionalProperties"].pop("anyOf", [])
-            for format in objects_to_check["additionalProperties"]["oneOf"]:
-                for key in format["properties"]:
-                    object_property = format["properties"][key]
-                    if "allOf" in object_property and "enum" in object_property["allOf"][0]:
-                        object_property["enum"] = object_property["allOf"][0]["enum"]
-                        object_property.pop("allOf")
+        objects_to_check["oneOf"] = objects_to_check.pop("anyOf", [])
+        for format in objects_to_check["oneOf"]:
+            for key in format["properties"]:
+                object_property = format["properties"][key]
+                if "allOf" in object_property and "enum" in object_property["allOf"][0]:
+                    object_property["enum"] = object_property["allOf"][0]["enum"]
+                    object_property.pop("allOf")
 
-        properties_to_change = ["primary_key", "input_schema"]
+        properties_to_change = ["primary_key", "validation_policy"]
         for property_to_change in properties_to_change:
-            schema["properties"]["streams"]["items"]["properties"][property_to_change]["oneOf"] = schema["properties"]["streams"]["items"][
-                "properties"
-            ][property_to_change].pop("anyOf")
-        return schema
-
-    @staticmethod
-    def add_legacy_format(schema: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
at this point since we are back to only supporting one format and not the map, there is no distinction between the shape of the config that requires transforming.